### PR TITLE
Log drains list command

### LINKED ISSUE #553: List existing drains
List existing drains by application 
```bash
scalingo --app my-app logs-drains
```

### PR DESCRIPTION
**Pull request by curzolapierre**
- Add list command
- Add a vsocde config file to debug CLI

Example
```bash
$ scalingo log-drains --app my-app
+------------------------------------+
|                 URL                |
+------------------------------------+
| ovh://:id@region.logs.ovh.com:6514 |
| tcp+tls://localhost:10303          |
+------------------------------------+
```

Fix #553

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (EtienneM) [inline .vscode/launch.json:1]: Like in go-scalingo, please remove this folder from the repository

File: cmd/commands.go [hunk 2]
```diff
@@ -218,6 +218,9 @@ var (
 		migrationListCommand,
 		migrationFollowCommand,
 
+		// Log drains
+		LogDrainsListCommand,
```
Comment (EtienneM) [inline cmd/commands.go:222]:
```suggestion
		logDrainsListCommand,
```

There is no reason for this function to be public

File: cmd/log_drains.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+			if len(c.Args()) == 0 {
+				err = log_drains.List(currentApp)
+			} else {
+				cli.ShowCommandHelp(c, "log-drains")
+			}
+
+			if err != nil {
+				errorQuit(err)
+			}
+		},
```
Comment (EtienneM) [inline cmd/log_drains.go:33]: Missing `BashComplete`:

```go
BashComplete: func(c *cli.Context) {
...
}
```

File: cmd/log_drains.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    # See also commands 'log-drains-add', 'log-drains-remove' and 'log-drains-info'`,
+
+		Action: func(c *cli.Context) {
+			currentApp := appdetect.CurrentApp(c)
+			var err error
+			if len(c.Args()) == 0 {
+				err = log_drains.List(currentApp)
+			} else {
+				cli.ShowCommandHelp(c, "log-drains")
```
Comment (EtienneM) [inline cmd/log_drains.go:27]: Please get rid of this case as early as possible. Hence the first thing to do in this function is:

```go
			if len(c.Args()) != 0 {
				cli.ShowCommandHelp(c, "log-drains")
				return
			}
```

File: cmd/log_drains.go [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+	LogDrainsListCommand = cli.Command{
+		Name:     "log-drains",
+		Category: "Log drains",
+		Flags:    []cli.Flag{appFlag},
+		Usage:    "List the log drains of an application",
+		Description: `List all the log drains of an application:
+
+    $ scalingo --app my-app log-drains
+
+    # See also commands 'log-drains-add', 'log-drains-remove' and 'log-drains-info'`,
```
Comment (EtienneM) [inline cmd/log_drains.go:19]: I think the `log-drains-info` will never exists

File: log_drains/list.go [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+func List(app string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	logDrains, err := c.LogDrainsList(app)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
```
Comment (EtienneM) [inline log_drains/list.go:20]: Don't use `errgo.Mask` but rather `errgo.Notef`

File: log_drains/list.go [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+	}
+
+	t := tablewriter.NewWriter(os.Stdout)
+
+	t.SetHeader([]string{"appID", "URL"})
```
Comment (EtienneM) [inline log_drains/list.go:25]:
```suggestion
	t.SetHeader([]string{"URL"})
```

Not sure it's useful to display the app ID here

The Notion section "New CLI Commands" displays an example of wanted output. I think only the URL can currently be displayed.

File: CHANGELOG.md [hunk 8]
```diff
@@ -7,6 +7,20 @@
 * Region Migration: Add instructions to change local Git URL at the end of the migration [#551](https://github.com/Scalingo/cli/pull/551)
 * Bugfix: fix support for `addon_updated` and `start_region_migration` event type [#558](https://github.com/Scalingo/cli/pull/558)
 * Bugfix: fix author of `restart` and `edit_variable` when it's an addon [#558](https://github.com/Scalingo/cli/pull/558)
+* Add log drains list command
+  [#560](https://github.com/Scalingo/cli/pull/560)
+  [go-scalingo #163](https://github.com/Scalingo/go-scalingo/pull/163)
+
+  Example:
```
Comment (EtienneM) [inline CHANGELOG.md:14]: The changelog does not contain examples

Comment (curzolapierre) [reply CHANGELOG.md:14]: It does actually, isn't it ?
But I understand that it should not :)

**Pull request merged**